.github/workflows/server-ci.yml

```
@@ -70,6 +66,10 @@ jobs:
         run: |
           mvn clean compile -U -Dmaven.javadoc.skip=true -ntp
 
+      - name: Prepare backend unique environment
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:41: Double quote to prevent globbing and word splitting

(shellcheck)